packages/zero-cache/src/auth/write-authorizer.ts

```diff
@@ -436,13 +428,17 @@ export class WriteAuthorizerImpl implements WriteAuthorizer {
     return true;
   }
 
+  /**
+   * Defaults to *false* if the policy is empty. At least one rule has to pass
+   * for the policy to pass.
+   */
   #passesPolicy(
     policy: Policy | undefined,
```

Comment:
cell policies also come in through here which have the opposite default.

Reply:
Yeah but the caller handles this by only calling `passesPolicy` in the cases where a cell policy actually exists.